tests/test_resources/test_clusters/test_cluster.py

```diff
@@ -102,6 +102,20 @@ def run_node_all(cmd):
     return rh.here.run(cmd, node="all")
 
 
+def sort_env_servlet_processes(env_servlet_processes: dict):
```

Comment:
why do we need to sort for the tests?

Reply:
The `env_servlet_processes` that are retrieved from the status get endpoint ordered differently than the expected `env_servlet_processes`. To make the comparison between them easier (just run `get_status_data["env_servlet_processes"] == env_servlet_processes`), we need to sort the env_servlet_processes by their keys.